git/README.md

```diff
@@ -59,3 +59,8 @@ We use widely established conventions for idiomatic Git commit messages:
 - Wrap the body at 72 characters
 
 [Read more &#x2197;](https://chris.beams.io/posts/git-commit/)
+
+## Releases
+
+For go projects, release tags follow this convention: `v[major].[minor].[patch]`: a semantic version with a `v` prefix for `gomod` to resolve versions properly.
```

Comment:
Does it apply just for go libraries-kind repos which we use as dependencies? Or project-kind repos also?

Reply:
At least for node, we have versions without `v`. Are we just keeping them and starting a new approach for the next release?

Reply:
Let's start with library repos first, but ultimately we shall have it for the node as well - it may be used as a library.